applications/EigenSolversApplication/tests/test_eigen_direct_solver.py

```diff
@@ -15,15 +16,23 @@ def _execute_eigen_direct_solver_test(self, class_name, solver_type):
         space = KratosMultiphysics.UblasSparseSpace()
 
         settings = KratosMultiphysics.Parameters('{ "solver_type" : "' + solver_type + '" }')
-        
+
         solver = ConstructSolver(settings)
 
         a = KratosMultiphysics.CompressedMatrix()
-        
-        KratosMultiphysics.ReadMatrixMarketMatrix('../../../kratos/tests/A.mm', a) # symmetric test matrix
-        
+
+        this_file_dir = os.path.dirname(os.path.realpath(__file__))
+        base_dir = os.path.dirname(os.path.dirname(os.path.dirname(this_file_dir)))
+        matrix_file_path = os.path.join(base_dir, "kratos")
```

Comment:
the following would also work:
`matrix_file_path = base_dir + os.sep + "kratos + os.sep + "tests" + os.sep + "A.mm"`

Reply:
or 
`matrix_file_path = os.path.join(base_dir, "kratos", "tests", "A.mm")`

Reply:
i like the one line option with join